test/test_functional.py

```diff
@@ -191,6 +191,69 @@ def test_istft_of_sine(self):
         self._test_istft_of_sine(amplitude=80, L=9, n=6)
         self._test_istft_of_sine(amplitude=99, L=10, n=7)
 
+    def _test_linearity_of_istft(self, data_size, kwargs):
+        for i in range(self.number_of_trials):
+            tensor1 = common_utils.random_float_tensor(i, data_size)
+            tensor2 = common_utils.random_float_tensor(i * 2, data_size)
+            a, b = torch.rand(2)
+            istft1 = torchaudio.functional.istft(tensor1, **kwargs)
+            istft2 = torchaudio.functional.istft(tensor2, **kwargs)
+            istft = a * istft1 + b * istft2
+            estimate = torchaudio.functional.istft(a * tensor1 + b * tensor2,  **kwargs)
+            self._compare_estimate(istft, estimate)
+
+    def test_linearity_of_istft1(self):
+        # hann_window, centered, normalized, onesided
+        kwargs1 = {
+            'n_fft': 12,
+            'window': torch.hann_window(12),
+            'center': True,
+            'pad_mode': 'reflect',
+            'normalized': True,
+            'onesided': True,
+        }
+        data_size = (2, 7, 7, 2)
+        self._test_linearity_of_istft(data_size, kwargs1)
+
+    def test_linearity_of_istft2(self):
+        # hann_window, centered, not normalized, not onesided
+        kwargs2 = {
+            'n_fft': 12,
+            'window': torch.hann_window(12),
+            'center': True,
+            'pad_mode': 'reflect',
+            'normalized': False,
+            'onesided': False,
+        }
+        data_size = (2, 12, 7, 2)
+        self._test_linearity_of_istft(data_size, kwargs2)
+
+    def test_linearity_of_istft3(self):
+        # hamming_window, centered, normalized, not onesided
+        kwargs3 = {
+            'n_fft': 12,
+            'window': torch.hamming_window(12),
+            'center': True,
+            'pad_mode': 'constant',
+            'normalized': True,
+            'onesided': False,
+        }
+        data_size = (2, 12, 7, 2)
+        self._test_linearity_of_istft(data_size, kwargs3)
+    #
```

Comment:
I'm assuming you pushed this to force a refresh of the test. Please mark it as "TODO: Remove this line after the tests are all passed."